promql/functions.go

```diff
@@ -1418,16 +1417,23 @@ func funcHistogramQuantile(vals []parser.Value, args parser.Expressions, enh *Ev
 		})
 	}
 
+	// Now do classic histograms that have already been filtered for conflicting native histograms
```

Comment:
Nit: End sentence with a period.
```suggestion
	// Now do classic histograms that have already been filtered for conflicting native histograms.
```